addons/account_edi_ubl_cii/wizard/account_move_send.py

```diff
@@ -84,14 +84,17 @@ def _compute_ubl_warnings(self):
             not_configured_partners = wizard.move_ids.partner_id.commercial_partner_id.filtered(
                 lambda partner: not (partner.peppol_eas and partner.peppol_endpoint)
             )
-            if len(not_configured_partners) == 1:
-                wizard.ubl_partner_warning = _("This partner is missing Peppol EAS or Peppol Endpoint field. "
-                                        "Please check those in its Accounting tab or the generated file will be incomplete.")
-            if len(not_configured_partners) > 1:
-                names = ', '.join(not_configured_partners[:5].mapped('display_name'))
-                wizard.ubl_partner_warning = _("The following partners are missing Peppol EAS or Peppol Endpoint field: %s. "
-                                        "Please check those in their Accounting tab. "
-                                        "Otherwise, the generated files will be incomplete.", names)
+            if not_configured_partners:
```

Comment:
Not directly related to this diff, but it seems we already have 3 different places where we're checking if a partner doesn't have a peppol_eas and a peppol_endpoint, and non stored compute function as helper could be useful in the future :nerd_face: 

[1](https://github.com/odoo/odoo/blob/3c48a062ada50c8c1d9257e08fa8f83a6a41fff5/addons/account_edi_ubl_cii/wizard/account_move_send.py#L83), [2](https://github.com/odoo/odoo/blob/2d33611e9c62a664c8d460adee99d25e42a37712/addons/account_peppol/models/res_partner.py#L93), [3](https://github.com/odoo/odoo/blob/3c48a062ada50c8c1d9257e08fa8f83a6a41fff5/addons/account_edi_ubl_cii/wizard/account_move_send.py#L85)

Reply:
yep, maybe eventually we will :slightly_smiling_face: